### ISSUE TITLE
Not compatible with Python 2.7.13 

### ISSUE THREAD

**Issue by roluke18** [conda/conda#4357]
(C:\Program Files (x86)\Anaconda2) C:\Users\Lucian>conda env list
An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Traceback (most recent call last):
  File "C:\Program Files (x86)\Anaconda2\Scripts\conda-script.py", line 5, in <module>
    sys.exit(conda.cli.main())
  File "C:\Program Files (x86)\Anaconda2\lib\site-packages\conda\cli\main.py", line 150, in main
    return conda_exception_handler(_main)
  File "C:\Program Files (x86)\Anaconda2\lib\site-packages\conda\exceptions.py", line 491, in conda_
exception_handler
    print_unexpected_error_message(e)
  File "C:\Program Files (x86)\Anaconda2\lib\site-packages\conda\exceptions.py", line 437, in print_
unexpected_error_message
    info_stdout, info_stderr = get_info()
  File "C:\Program Files (x86)\Anaconda2\lib\site-packages\conda\exceptions.py", line 407, in get_in
fo
    args.func(args, p)
  File "C:\Program Files (x86)\Anaconda2\lib\site-packages\conda\cli\main_info.py", line 155, in exe
cute
    from conda.api import get_index
  File "C:\Program Files (x86)\Anaconda2\lib\site-packages\conda\api.py", line 7, in <module>
    from .fetch import fetch_index
  File "C:\Program Files (x86)\Anaconda2\lib\site-packages\conda\fetch.py", line 27, in <module>
    from .connection import CondaSession, RETRIES
  File "C:\Program Files (x86)\Anaconda2\lib\site-packages\conda\connection.py", line 58, in <module
>
    python=platform.python_implementation(),
  File "C:\python27\Lib\platform.py", line 1460, in python_implementation
    return _sys_version()[0]
  File "C:\python27\Lib\platform.py", line 1422, in _sys_version
    repr(sys_version))
ValueError: failed to parse CPython sys.version: '2.7.12 |Anaconda 4.2.0 (32-bit)| (default, Jun 29
2016, 11:42:13) [MSC v.1500 32 bit (Intel)]'

**Comment by kalefranz** [comment]
What version of conda are you running?  No way it's recent I don't think.

    conda update conda

**Comment by roluke18** [comment]
Anaconda3 4.2.0. I run Win 7 64-bits. At the time of the install i had Python 2.7.13 installed and Anaconda didn't work.
So far I tried to install Python 2 and 3 and after install Anaconda. None of it worked. I installed only Anaconda 3 and it works; i can also start Python 3 in my shell.The thing is Anaconda 4.2.0 doesn't work if you have Python previously installed. (no Prompt, no Jupyter, no Spyder)
Interesting fact: I've created a python 2 environment and a python 3 environment. In py2, when i type python, it starts python 2.7.13. But in Jupyter i have no Python 2 kernel, only default and conda root.
Lucian
